packages/now-cli/src/util/env/get-env-records.ts

```diff
@@ -1,69 +1,24 @@
 import { Output } from '../output';
 import Client from '../client';
-import {
-  ProjectEnvVariable,
-  ProjectEnvTarget,
-  PaginationOptions,
-} from '../../types';
+import { ProjectEnvVariable, ProjectEnvTarget } from '../../types';
 import { URLSearchParams } from 'url';
 
-type ApiVersion = 4 | 5;
-
-type APIV4Response = ProjectEnvVariable[];
-
-interface APIV5Response {
-  pagination: PaginationOptions;
-  envs: ProjectEnvVariable[];
-}
-
 export default async function getEnvVariables(
   output: Output,
   client: Client,
   projectId: string,
-  apiVersion: 4,
   target?: ProjectEnvTarget
-): Promise<APIV4Response>;
-
-export default async function getEnvVariables(
-  output: Output,
-  client: Client,
-  projectId: string,
-  apiVersion: 5,
-  target?: ProjectEnvTarget,
-  next?: number
-): Promise<APIV5Response>;
-
-export default async function getEnvVariables<V extends ApiVersion>(
-  output: Output,
-  client: Client,
-  projectId: string,
-  apiVersion: V,
-  target?: ProjectEnvTarget,
-  next?: number
 ) {
   output.debug(
     `Fetching Environment Variables of project ${projectId} and target ${target}`
   );
   const query = new URLSearchParams();
-  if (apiVersion >= 5) {
-    query.set('limit', String(20));
-  }
```

Comment:
What happened to `limit`?

Reply:
v6 doesn't have pagination anymore  (it returns all env variables, which are capped to 100 env variables).